<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="trangntt-w7" name="Default test" tests="3" failures="0" timestamp="13 Apr 2016 10:30:48 GMT" time="1.338" errors="0">
  <testcase name="@BeforeClass setupClass" time="0.0" classname="com.vsii.tsc.guru.testcase.NewCustomer">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.vsii.tsc.guru.commethods.CommonMethods.checkLogin(CommonMethods.java:13)
	at com.vsii.tsc.guru.testcase.NewCustomer.setupClass(NewCustomer.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setupClass -->
  <testcase name="@AfterMethod afterMethod" time="0.0" classname="com.vsii.tsc.guru.testcase.NewCustomer">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@AfterMethod afterMethod" time="0.0" classname="com.vsii.tsc.guru.testcase.NewCustomer">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@AfterMethod afterMethod" time="0.0" classname="com.vsii.tsc.guru.testcase.NewCustomer">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="CN01" time="0.001" classname="com.vsii.tsc.guru.testcase.NewCustomer">
    <skipped/>
  </testcase> <!-- CN01 -->
  <testcase name="CN02" time="0.001" classname="com.vsii.tsc.guru.testcase.NewCustomer">
    <skipped/>
  </testcase> <!-- CN02 -->
  <testcase name="CN03" time="0.001" classname="com.vsii.tsc.guru.testcase.NewCustomer">
    <skipped/>
  </testcase> <!-- CN03 -->
</testsuite> <!-- Default test -->
